docs/guides/mocking.mdx

```diff
@@ -28,6 +34,40 @@ class UserTile extends StatelessWidget {
 }
 ```
 
+For this example, the Widget tree for your `HomePage` widget in your app might look like this:
+
+```tree
+App
+└── UserProvider
+    └── HomePage
+        └── UserTile
+            └── Consumer<UserProvider>
+                └── Text
+```
+
+## Problem
+
+If the `UserTile` is cataloged using the following code
+
+```dart
+Widget userTileUseCase(BuildContext context) {
+  return UserTile();
+}
+```
+
+Flutter throws an error indicating that a `UserProvider` is missing from the Widget tree, as the `Consumer` within the `UserTile` depends on the `UserProvider`.
+
+```tree
+WidgetbookApp
+└── UserTile
+    └── Consumer<UserProvider>         ❌ Error: UserProvider not found
+        └── Text
+```
+
+To catalog the `UserTile` widget, you need to
+1. remove the dependency of the `UserTile` by using property extraction or
+1. provide the `UserProvider` to the Widget tree. 
+
```

Comment:
**suggestion:** the "or" at the end of point one, is a bit confusing. We can rephrase as follows

```suggestion
To catalog the `UserTile` widget, you need to do **one of the following**:
1. Remove the dependency of the `UserTile` by using property extraction.
1. Provide the `UserProvider` to the Widget tree. 

```